### PR TITLE
ZenGo new member blog post

### PR DESCRIPTION
**Pull request by jpgross3**
waiting till Monday (12 April) to post. hoping to get a more thorough description of ZenGo.

### CODE REVIEW

**Comment by jpgross3** [comment]
@alanshaw I'll let you merge this PR if it looks good to you. 

cc: @jsoares

**Review by alanshaw — APPROVED** [review]

**Pull request merged**
